src/cmd/cli/command/commands.go

```diff
@@ -989,7 +981,7 @@ var deploymentsCmd = &cobra.Command{
 	Args:        cobra.NoArgs,
 	Short:       "List deployments",
```

Comment:
Let's update this description to something which indicates how this is different than `defang deployments list`.